src/pkg/clouds/gcp/logging.go

```diff
@@ -21,25 +22,33 @@ func (gcp Gcp) NewTailer(ctx context.Context) (*Tailer, error) {
 	if err != nil {
 		return nil, err
 	}
+	projectIds = append([]string{gcp.ProjectId}, projectIds...)
```

Comment:
I think this is a bit weird. Either remove `ProjectId` from `Gcp` struct, or make it an array.